### PR TITLE
chore: Create the Initial CODEOWNERS file

### LINKED ISSUE #6074: Create the Initial CODEOWNERS file
Create CODEOWNERS file and related teams on a github.

### PR DESCRIPTION
**Pull request by dunqan**
Create CODEOWNERS file and related teams on a github.

Closes #6074

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+#  znikola - Nikola Zarić
+#  developpeurweb - Miguel Estrada
+
+
+
+# ===========================================================
+#  @SAP/spartacus-global-owners
+# ===========================================================
+#  Used for approving minor changes, large-scale refactorings, and emergency situations.
+#  (secret team to avoid review requests, it also doesn't inherit from @angular/framework because nested teams can't be secret)
```
Comment (marlass) [inline .github/CODEOWNERS]: 👀

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+# ================================================
+#  Routing Functionality
+# ================================================
+/projects/core/src/routing/**                                            @SAP/spartacus-routing-owners @SAP/spartacus-global-owners
+
+# ================================================
+#  CODEOWNERS Owners owners ...
+# ================================================
+
+/.github/CODEOWNERS                                                      @dunqan @SAP/spartacus-global-approvers
```
Comment (marlass) [inline .github/CODEOWNERS]:
```suggestion
/.github/CODEOWNERS                                                      @dunqan @SAP/spartacus-global-owners
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+/projects/core/src/cart/**                                               @SAP/spartacus-cart-owners @SAP/spartacus-global-owners
+/projects/storefrontlib/src/cms-components/cart/**                       @SAP/spartacus-cart-owners @SAP/spartacus-global-owners
+
+# ================================================
+#  Routing Functionality
+# ================================================
+/projects/core/src/routing/**                                            @SAP/spartacus-routing-owners @SAP/spartacus-global-owners
+
+# ================================================
+#  CODEOWNERS Owners owners ...
```
Comment (marlass) [inline .github/CODEOWNERS]: ;)

File: .github/CODEOWNERS (hunk 1 above)
Comment (dunqan) [reply .github/CODEOWNERS]: ;-)

**Pull request merged**